Week-3/autoLoan/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Comparison of Offers</title>
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+<div class="container">
+    <h1>Comparison of Dealer Incentives and Loan Offers</h1>
+    <div class="offer">
+        <div class="offerTitle">Purchase Offer 1</div>
+        <div class="offerContents">
+            <!-- Content for Offer 1 -->
+            <table>
+                <tr><th>Item</th><th>Cost</th></tr>
```

Comment:
Ensure accessibility by using <th scope="col"> or <th scope="row"> to define header cells in tables, providing context for screen readers.